connectors-home/modules/ROOT/pages/introduction/intro-config-oauth2.adoc

```diff
@@ -0,0 +1,189 @@
+| Scopes | String | Space-delimited list of OAuth scopes to request during the OAuth dance. This value defaults to the scopes in the annotation. | `offline_access` |
+| Resource Owner ID | String | ID of the owner's user or account authorizing the connector to access their resources. | |
+|===
+
+*OAuth 2.0 Callback Configuration:*
+
+[%header,cols="20s,20a,35a,20a,5a"]
+|===
+| Name | Type | Description | Default Value | Required
+| Callback Path | String | Path on your server in which the user's browser redirects to when authorization is successful with the OAuth provider. The callback path is typically `/callback`. This configuration uses your *HTTP Listener* setup to build the callback path, for example, `\http://0.0.0.0:{port}/callback`. | | X
+| Authorize Path | String | Path on your server in which the user begins the OAuth dance. The authorize path is typically `/authorize`. | | X
+| External Callback Url | String | Full, publicly accessible URL on your server in which the user's browser is redirected to when authorization is successful with the OAuth provider. This field enables you to override the *Callback path* field, for example, `\http://my.server.com/callback`. | |
+|===
+
+== OAuth 2.0 Flow Overview
+
+Anypoint Connectors use the Authorization Code Grant flow for their OAuth 2.0 dance. The following steps show a typical flow for configuring OAuth 2.0 for a connector:
+
+. In your browser, navigate to the path specified in the *Authorize Path* field to begin the OAuth dance. For example, if you are running the connector on `localhost` and your path is set to `/authorize` with port `8080`, navigate to `\http://localhost:8080/authorize`.
+. Provide values for *Authorization URL*, *Scopes*, *Resource Owner ID*, and *Callback Path*. The following conditions must be true for the OAuth provider to accept the authorization request:
++
+* The *Authorization URL* must be reachable from the user's browser.
+* The provided scopes for *Scopes* are supported.
+* The *Resource Owner ID* (if provided) must be registered with the OAuth provider.
+* The full *Callback Path* must be registered with the OAuth provider.
+. Enter your username, password, and other credentials that the OAuth provider accepts, such as 2FA requests. If the authentication is successful, the OAuth provider might ask permission for the application to access your account. This typically occurs the first time you authenticate.
+. Once the OAuth provider authenticates the user and authorizes access to the connector, the user's web browser is redirected to the callback URL that is registered with the OAuth provider, for example, `\http://localhost:8080/callback`.
++
+Ensure that the user running the flow has access to that server and endpoint, otherwise the authorization approval does not reach the connector.
```

Comment:
```suggestion
Ensure that the user running the flow has access to that server and endpoint, otherwise, the authorization approval does not reach the connector.
```

Reply:
What server is this referring to?

This is a case where I made it future tense. The "does not reach" sounds wrong to me, but feel free to disagree.